### PR TITLE
Email Validation not available in Class Definitions

### PR DESCRIPTION
**Pull request by nderguti**
https://synjira.atlassian.net/browse/CELDEV-477

### CODE REVIEW

File: src/main/java/com/celements/mailsender/CelMailSenderService.java [hunk 1]
```diff
@@ -14,6 +14,12 @@
 
   private IMailObjectRole injectedMailObject;
 
+  public static final String EMAIL_VALIDATION_REGEX = "^[\\w\\-\\u0153\\u00E0-\\u00F6\\u00F8-\\u00FF.+]+[@][\\w\\-\""
+      + "\u0153\\u00E0-\\u00F6\\u00F8-\\u00FF]+([.]([\\w\\-\\u0153\\u00E0-\\u00F6\\u00F8-\\u00FF]+))+$";
+
+  public static final String EMAIL_VALIDATION_REGEX_CLASS_DEFINITIONS = "/^[\\w\\-\\u0153\\u00E0-\\u00F6\\u00F8-\\u00FF.+]+[@][\\w\\-\""
```
Comment (ebeutler) [inline src/main/java/com/celements/mailsender/CelMailSenderService.java]: Use EMAIL_VALIDATION_REGEX when building EMAIL_VALIDATION_REGEX_CLASS_DEFINITIONS

File: src/main/java/com/celements/mailsender/CelMailSenderService.java [hunk 2]
```diff
@@ -61,14 +67,11 @@ IMailObjectRole getNewMailObject() {
 
   @Override
   public String getEmailValidationRegex() {
-    // chars - a-z A-Z 0-9 œ à á â ã ä å æ ç è é ê ë ì í î ï ð ñ ò ó ô õ ö ø ù ú û ü ý þ ÿ
```
Comment (ebeutler) [inline src/main/java/com/celements/mailsender/CelMailSenderService.java:64]: Add this comment to EMAIL_VALIDATION_REGEX. It's much more readable than "\\w\\-\\u0153\\u00E0-\\u00F6\\u00F8-\\u00FF" if out want to no which characters are allowed.

**Review by ebeutler — CHANGES_REQUESTED** [review]

**Review by msladek — CHANGES_REQUESTED** [review]
agree with @ebeutler

File: src/main/java/com/celements/mailsender/CelMailSenderService.java [hunk 3]
```diff
@@ -62,13 +68,11 @@ IMailObjectRole getNewMailObject() {
   @Override
   public String getEmailValidationRegex() {
     // chars - a-z A-Z 0-9 œ à á â ã ä å æ ç è é ê ë ì í î ï ð ñ ò ó ô õ ö ø ù ú û ü ý þ ÿ
```
Comment (ebeutler) [inline src/main/java/com/celements/mailsender/CelMailSenderService.java]: please move the commen to the constant (around line 17)

**Review by ebeutler — CHANGES_REQUESTED** [review]

**Review by ebeutler — APPROVED** [review]

**Review by msladek — APPROVED** [review]

**Pull request merged**